<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hqc application</string>
  </property>
  <property name="windowIcon">
  <iconset>icons:hqc_logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupNewWindow">
      <property name="title">
       <string>Tools</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonSearchTool">
         <property name="text">
          <string>&amp;Search Tool</string>
         </property>
         <property name="icon">
         <iconset>icons:search_red.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonTextData">
         <property name="text">
          <string>&amp;Text Data</string>
         </property>
         <property name="icon">
         <iconset>icons:textdata.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonWatchRR">
         <property name="text">
          <string>&amp;WatchRR</string>
         </property>
         <property name="icon">
         <iconset>icons:watchrr.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonRejected">
         <property name="text">
          <string>&amp;Rejected</string>
         </property>
         <property name="icon">
         <iconset>icons:rejectedobs.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupChanges">
      <property name="title">
       <string>Changes</string>
      </property>
      <layout class="QGridLayout" name="gridChanges" rowstretch="0">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="buttonUndo">
         <property name="text">
          <string>Undo</string>
         </property>
         <property name="icon">
         <iconset>icons:undo.svg</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Z</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="buttonRedo">
         <property name="text">
          <string>Redo</string>
         </property>
         <property name="icon">
         <iconset>icons:redo.svg</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+Z</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>118</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Save…</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenKro"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionUserManual"/>
    <addaction name="actionNews"/>
    <addaction name="actionFlags"/>
    <addaction name="actionParameters"/>
    <addaction name="separator"/>
    <addaction name="actionAboutHqc"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSearchTool"/>
    <addaction name="actionTextData"/>
    <addaction name="actionWatchRR"/>
    <addaction name="actionRejected"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionUserManual">
   <property name="text">
    <string>&amp;User manual…</string>
   </property>
  </action>
  <action name="actionFlags">
   <property name="text">
    <string>&amp;Flags…</string>
   </property>
  </action>
  <action name="actionNews">
   <property name="text">
    <string>&amp;News…</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="text">
    <string>&amp;Parameters…</string>
   </property>
  </action>
  <action name="actionAboutHqc">
   <property name="text">
    <string>&amp;About HQC</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionOpenKro">
   <property name="text">
    <string>Open Kro…</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings…</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
   <iconset>icons:undo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
   <iconset>icons:redo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionSearchTool">
   <property name="icon">
    <iconset>icons:search_red.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Search Tool</string>
   </property>
   <property name="toolTip">
    <string>Start search tool</string>
   </property>
  </action>
  <action name="actionTextData">
   <property name="icon">
    <iconset>icons:textdata.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Text Data</string>
   </property>
   <property name="toolTip">
    <string>Start text data tool</string>
   </property>
  </action>
  <action name="actionWatchRR">
   <property name="icon">
    <iconset>icons:watchrr.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;WatchRR</string>
   </property>
   <property name="toolTip">
    <string>Start WatchRR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRejected">
   <property name="icon">
    <iconset>icons:rejectedobs.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Rejected</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionSearchTool</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onNewSearch()</slot>
  </connection>
  <connection>
   <sender>actionWatchRR</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onNewWatchRR()</slot>
  </connection>
  <connection>
   <sender>actionRejected</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onNewRejectedObs()</slot>
  </connection>
  <connection>
   <sender>actionTextData</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onNewTextdata()</slot>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutHqc</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onAboutHqc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onAboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenKro</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onStartKro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFlags</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onHelpFlag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNews</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onHelpNews()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionParameters</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onHelpParam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onRedoChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onUndoChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onSaveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUserManual</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onHelpUse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>AppWindow</receiver>
   <slot>onUserSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNewSearch()</slot>
  <slot>onNewTextdata()</slot>
  <slot>onNewWatchRR()</slot>
  <slot>onNewRejectedObs()</slot>
  <slot>onSaveChanges()</slot>
  <slot>onUndoChanges()</slot>
  <slot>onRedoChanges()</slot>
  <slot>onHelpUse()</slot>
  <slot>onHelpNews()</slot>
  <slot>onHelpFlag()</slot>
  <slot>onHelpParam()</slot>
  <slot>onAboutHqc()</slot>
  <slot>onAboutQt()</slot>
  <slot>onStartKro()</slot>
  <slot>onUserSettings()</slot>
 </slots>
</ui>
